sky/shell/platform/mojo/dart_tracing.cc

```diff
@@ -0,0 +1,172 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
This file is basically the same as the one in Mojo, right? Is there a way to share the code?